.github/workflows/ci.yml

```diff
@@ -2,10 +2,20 @@
 name: CI
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
**praises:** nice

I think that may be the root cause of previous issues not catched by the CI